src/components/SelectListForm.jsx

```diff
@@ -36,11 +40,6 @@ export default function SelectListForm({
 					<button onClick={() => navigate('/list')}>View List</button>
 				</div>
 			)}
-			<div>
-				<h3>
```

Comment:
I think we want to keep this h3 inside the div and where it is originally. this is according to the lofi mockup layout.